.github/actions/action-test/action.yaml

```diff
@@ -9,6 +14,23 @@ runs:
     - shell: bash
       run: npm run prepare
     - uses: ./versions
+      if: inputs.use-versions == 'true'
+      with:
+        go: 1.17.6
+        java: 17.0.2-tem
+        node: 16.13.2
+        python: 3.10.2
+        terraform: 1.1.5
+    - shell: bash
+      run: |
+        echo "java=17.0.2-tem" > .sdkmanrc
+        echo "16.13.2" > .node-version
+        echo "3.10.2" > .python-version
+        echo "1.17.6" > .go-version
+        echo "1.1.5" > .terraform-version
+      if: inputs.use-versions != 'true'
+    - uses: ./
+      if: inputs.use-versions != 'true'
```

Comment:
@shakefu I decided it was good to add this test so we tested both cases.  

Of course it works here.  :(